fcc-test-suite-template-master/index.html

```diff
@@ -7,6 +7,7 @@
 	<title>Tribute to Steve Jobs</title>
```

Comment:
- [ ] This PR should be invalidated, kindly if the student didn't add the description and fix linters, invalidate the PR